Detectors/TPC/reconstruction/run/testRawReader.cxx

```diff
@@ -0,0 +1,95 @@
+///
+/// @file   testRawReader.cxx
```

Comment:
is this supposed to be a unit test? Can it fail/pass? If yes, we should add it to the ctests.